src/components/sbb-toast/sbb-toast.scss

```diff
@@ -0,0 +1,140 @@
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include host-component-properties;
+
+:host {
+  --toast-color-default: var(--sbb-color-white-default);
+  --toast-background-color: var(--sbb-color-midnight-default);
+  --toast-border-radius: var(--sbb-border-radius-4x);
+  --toast-gap: var(--sbb-spacing-fixed-2x);
+  --toast-padding-block: var(--sbb-spacing-responsive-xxxs);
+  --toast-padding-inline: var(--sbb-spacing-responsive-xs);
+  --toast-margin: var(--sbb-spacing-responsive-s);
+  --toast-max-width: #{px-to-rem-build(400)};
+  --toast-width: #{calc(100% - (2 * var(--sbb-spacing-responsive-s-zero)))};
+}
+
+.overlay-hidden {
+  display: none;
+}
+
+.toast-wrapper {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100vw;
+  height: 100vh;
+  contain: content;
+  pointer-events: none;
+}
+
+.toast {
+  @include text-s--bold;
+  @include shadow-level-5-hard;
+  @include absoluteCenterX;
+
+  position: fixed;
+  padding-block: var(--toast-padding-block);
+  padding-inline: var(--toast-padding-inline);
+  margin: var(--toast-margin) 0;
+  width: calc(100% - (2 * var(--sbb-spacing-responsive-s-zero)));
+
+  @include mq($from: small) {
+    width: var(--toast-max-width);
+  }
+
+  display: flex;
+  flex-direction: row;
+  align-items: start;
+  inset-block-end: 0;
+  gap: var(--toast-gap);
+  background: var(--toast-background-color);
+  color: var(--toast-color-default);
+  border-radius: var(--toast-border-radius);
+
+  @include ifForcedColors {
+    border: #{px-to-rem-build(1)} solid windowText;
+  }
+}
+
+.toast-vertical-start {
+  inset-block-start: 0;
+  inset-block-end: auto;
+}
+
+.toast-horizontal-start {
+  inset-inline-start: var(--toast-margin);
+  transform: none;
+}
+
+.toast-horizontal-end {
+  inset-inline-start: auto;
+  inset-inline-end: var(--toast-margin);
+  transform: none;
+}
+
+.toast.toast--button-icon {
+  padding-inline-end: calc(
+    (2 * var(--sbb-spacing-fixed-2x)) + var(--sbb-size-button-m-min-height-small)
+  );
+  @include mq($from: medium) {
+    padding-inline-end: calc(
+      (2 * var(--sbb-spacing-fixed-2x)) + var(--sbb-size-button-m-min-height-medium)
+    );
+  }
+
+  .toast-spacer {
+    display: none;
+  }
+}
+
+.toast-text {
+  word-break: break-all;
```

Comment:
This layout on small screens does not look right. The message text should wrap to the words without breaking the letters, and the button maybe should have enough space not to break (?)

<img src="https://user-images.githubusercontent.com/101181211/187503825-785e4e68-2cb1-42dc-b59d-48c5cf677966.png" height="75">

```suggestion
  word-break: break-word;
```

Reply:
Text fixed & pushed;
about the button, I changed the word-break prop for sbb-link in commit cbc9d3b0529c56ebb7aa1159e3d4bfa88164109d, but then I reverted it because OFC it was not a solution..